src/pages/index.tsx

```diff
@@ -8,6 +8,7 @@ import { useEffect, useState } from "react";
 import type { User, Comment } from "@prisma/client";
 import { CreateBlogPostButton } from "~/components/create-blog-post-widget/CreateBlogPostButton";
```

Comment:
can we remove this import - in future changes?

Reply:
removed thanks for the catch